test/unit/behaviours/get-living-status.test.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const reqres = require('reqres');
+const Behaviour = require('../../../apps/right-to-rent-check/behaviours/get-living-status');
+
+describe('apps/right-to-rent-check/behaviours/get-living-status', () => {
+    it('exports a function', () => {
+      Behaviour.should.be.a('function');
+    });
+
+    class Base {
+      locals() {}
+    }
+    let req;
+    let res;
+    let sessionModel;
+    let instance;
+    let GetLivingStatus;
+    let translate;
+
+    const superLocals = {
+      tenants: [{'tenant-name': 'Mo Khan'}],
+    };
+
+    beforeEach(() => {
+      sessionModel = {
+        get: sinon.stub()
+      };
+      translate = sinon.stub();
+      res = reqres.res();
+      req = reqres.req({sessionModel, translate});
+      sinon.stub(Base.prototype, 'locals').returns(superLocals);
+      GetLivingStatus = Behaviour(Base);
+      instance = new GetLivingStatus();
+    });
+
+    afterEach(() => {
+      Base.prototype.locals.restore();
+    });
+
+    describe('locals()', () => {
+      describe('when a tenant is currently living in the property', () => {
+        const expected = {
+              tenants: [{'tenant-name': 'Mo Khan'}],
+              livingStatusHeader: 'yes living',
+              livingStatusId: 'yes-living-status',
+              livingStatusIntro: 'intro'
+            };
+        let result;
+        beforeEach(() => {
+          req.sessionModel.get.withArgs('living-status').returns('yes');
+          req.translate.withArgs('pages.request-another-tenant.yes-living-status.header').returns('yes living');
+          req.translate.withArgs('pages.request-another-tenant.yes-living-status.intro').returns('intro');
+          result = instance.locals(req, res);
+        });
+        it('has a yes living status', () => {
+            result.livingStatusId.should.equal(expected.livingStatusId);
```

Comment:
this is asserting your test set up `req.sessionModel.get.withArgs('living-status').returns('yes');`